scripts/cibuild/BuildValidator.ps1

```diff
@@ -51,6 +51,5 @@ if ($BuildRTM -eq 'false')
         & $NuGetValidator localization --artifacts-path $ArtifactsLocation --output-path $ArtifactsLogOutputDir --comments-path $LocalizationRepository
     }
 
-    # return the exit code from the validator
-    exit $LASTEXITCODE
+    exit 0
```

Comment:
Why is this change needed?

Reply:
The build validator used in here had a bug, or specifically a scenario that wasn't considered. 

I fixed the bug in the build validator, https://github.com/NuGet/Entropy/commit/3d8305664e39c8617693ca3c1b9dadfcfd8a0768#diff-6f2e63fc0107f857fa63344fa4d35772 and published a new version of that validator. I will update that and revert this change.

Reply:
Reverted and updated to the new version of the validator.